src/sentry/issues/issue_occurrence.py

```diff
@@ -34,6 +34,7 @@ class IssueOccurrenceData(TypedDict):
     detection_time: float
     level: str | None
     culprit: str | None
+    initial_issue_priority: int | None
```

Comment:
Should this be `NotRequired` rather than `None`? I assume that the key isn't going to be present in many cases